pkg/webhook/admission/servingruntime/servingruntime_webhook.go

```diff
@@ -39,6 +42,11 @@ const (
 	ProrityIsNotSameError                      = "different priorities assigned for the model format %s"
 	ProrityIsNotSameServingRuntimeError        = "%s under the servingruntime %s"
 	ProrityIsNotSameClusterServingRuntimeError = "%s under the clusterservingruntime %s"
+	InvalidWorkerSpecSizeValueError            = "the WorkerSpec.Size cannot be less than 2(%d)"
+	InvalidPipelineParallelSizeValueError      = "the PIPELINE_PARALLE_SIZE cannot be less than 2(%s)"
+	InvalidTensorParallelSizeValueError        = "the TENSOR_PARALLE_SIZE cannot be less than 1(%s)"
+	InvalidParallelSizeValueError              = "the value of PIPELINE_PARALLEL_SIZE or TENSOR_PARALLEL_SIZE is incorrect"
+	InvalidMultiNodeEnvVariablesError          = "the %s %s is invalid: %s"
```

Comment:
Are these constants in previously changed files that we can reuse here?

Reply:
For ServingRuntime, each message is duplicated for both ClusterServingRuntime and ServingRuntime. I attempted to create a generic error message to accommodate both scenarios, which results in a slightly different log format compared to the inference validation error message. Therefore, I believe we should avoid reusing the InferenceService error message in this case.